grc/core/FlowGraph.py

```diff
@@ -462,10 +474,12 @@ def verify_and_get_port(key, block, dir):
                 self.connect(source_port, sink_port)
 
         except (KeyError, LookupError) as e:
-            Messages.send_error_load(
-                'Connection between {}({}) and {}({}) could not be made.\n\t{}'.format(
-                    src_blk_id, src_port_id, snk_blk_id, snk_port_id, e))
-            had_connect_errors = True
+            # hide connection errors caused by missing view-only values
+            if self.view_only and not self.trigger_trust_prompt:
+                Messages.send_error_load(
+                    'Connection between {}({}) and {}({}) could not be made.\n\t{}'.format(
+                        src_blk_id, src_port_id, snk_blk_id, snk_port_id, e))
+                had_connect_errors = True
```

Comment:
Is this a left-over from the "best-effort" loading experiments?

Reply:
Yeah, good find!